src/proposals/20241114/SparkBase_20241114.t.sol

```diff
@@ -0,0 +1,130 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.10;
+
+import 'src/SparkTestBase.sol';
+
+import { CCTPForwarder } from "xchain-helpers/forwarders/CCTPForwarder.sol";
+
+import { IALMProxy }         from "spark-alm-controller/src/interfaces/IALMProxy.sol";
+import { IRateLimits }       from "spark-alm-controller/src/interfaces/IRateLimits.sol";
+import { ForeignController } from "spark-alm-controller/src/ForeignController.sol";
+import { RateLimitHelpers }  from "spark-alm-controller/src/RateLimitHelpers.sol";
+
+import { IPSM3 } from "spark-psm/src/interfaces/IPSM3.sol";
+
+contract SparkBase_20241114Test is SparkBaseTestBase {
+
+    address internal constant FREEZER = 0x90D8c80C028B4C09C0d8dcAab9bbB057F0513431;  // Gov. facilitator multisig
+    address internal constant RELAYER = 0x8a25A24EDE9482C4Fc0738F99611BE58F1c839AB;
+
+    address internal constant DEPLOYER = 0x6F3066538A648b9CFad0679DF0a7e40882A23AA4;
+
+    constructor() {
+        id = '20241114';
+    }
+
+    function setUp() public {
+        vm.createSelectFork(getChain('base').rpcUrl, 22015320);  // Nov 5, 2024
+        payload = deployPayload();
+    }
+
+    function testALMControllerDeployment() public {
+        // Copied from the init library, but no harm checking this here
+        IALMProxy         almProxy   = IALMProxy(Base.ALM_PROXY);
+        IRateLimits       rateLimits = IRateLimits(Base.ALM_RATE_LIMITS);
+        ForeignController controller = ForeignController(Base.ALM_CONTROLLER);
+
+        assertEq(almProxy.hasRole(0x0, Base.SPARK_EXECUTOR), true,   "incorrect-admin-almProxy");
```

Comment:
spacing is weird here, the next block aligns by address instead of by the start of the revert string

```suggestion
        assertEq(almProxy.hasRole(0x0,      Base.SPARK_EXECUTOR), true, "incorrect-admin-almProxy");
        assertEq(rateLimits.hasRole(0x0,    Base.SPARK_EXECUTOR), true, "incorrect-admin-rateLimits");
        assertEq(controller.hasRole(0x0,    Base.SPARK_EXECUTOR), true, "incorrect-admin-controller");
```

Reply:
it does still look weird, i understand the formatting criteria is aligning `,`s?
we could think of a formal formatter and would save a lot of nits

Reply:
another fix, there is no hard rule here other than improving readability